crawl-AR/test/parser.js

```diff
@@ -0,0 +1,57 @@
+let assert = require("assert");
+let parser = require('./../lib/parser');
+
+describe("Parser", function() {
+  it("Empty entry Returns empty index", async () => {
+    let result = parser.parse([]);
+    assert.deepEqual(await result, {
+      byAddress: [],
+       byGeolocation: []
```

Comment:
Nit: indentation here is +1 space from previous.  Is this a convention of yours or did it slip in?  Looks be happen in all following object literals.  I'm liberal on style conventions so fine either way.